core/src/error.rs

```diff
@@ -240,3 +245,29 @@ pub enum EventError {
 
 /// Represents the result of a contract call.
 pub type OdraResult<T> = Result<T, OdraError>;
+
+impl From<CLValueError> for OdraError {
+    fn from(error: CLValueError) -> Self {
+        match error {
+            CLValueError::Serialization(_) => OdraError::VmError(Serialization),
+            CLValueError::Type(cl_type_mismatch) => OdraError::VmError(VmError::TypeMismatch {
+                expected: cl_type_mismatch.expected.clone(),
+                found: cl_type_mismatch.found.clone()
+            })
+        }
+    }
+}
+
+impl From<BytesReprError> for OdraError {
+    fn from(error: BytesReprError) -> Self {
+        match error {
+            BytesReprError::EarlyEndOfStream => ExecutionError::EarlyEndOfStream.into(),
+            BytesReprError::Formatting => ExecutionError::Formatting.into(),
+            BytesReprError::LeftOverBytes => ExecutionError::LeftOverBytes.into(),
+            BytesReprError::OutOfMemory => ExecutionError::OutOfMemory.into(),
+            BytesReprError::NotRepresentable => ExecutionError::NotRepresentable.into(),
+            BytesReprError::ExceededRecursionDepth => ExecutionError::ExceededRecursionDepth.into(),
+            _ => ExecutionError::Formatting.into()
+        }
```

Comment:
You can have only one `into`
```suggestion
        }.into()
```